build/docker/marti-dev/app-files/root/etc/ssmtp/ssmtp.conf

```diff
@@ -0,0 +1,4 @@
+AuthMethod=LOGIN
+FromLineOverride=yes
+UseSTARTTLS=Yes
+UseTLS=Yes
```

Comment:
Could become problematic, although postfix etc. have a self signed certificate as a default certificate this will likely cause issues if not preconfigured manually.

Reply:
This option simply requires TLS when opening a connection from this host (i.e. `dice.tripleawarclub.org`) to the external mail relay specified in the `mailhub` option.  [SSMTP](https://wiki.archlinux.org/index.php/SSMTP) does not run a local mail relay--it is purely an SMTP client:

> SSMTP is a program which delivers email from a local computer to a configured mailhost (mailhub). It is not a mail server (like feature-rich mail server sendmail) and does not receive mail, expand aliases or manage a queue.

It would be nice if we could drop Postfix in production.  The new dice server will most likely use a Node SMTP module that does not require an external mail application like PHP does.  Does anything else on that host require a mail server to be running?

Reply:
Thanks for the clarification.
NodeBB does indeed not require a the postfix sendmail application, we can (and probably have to) use nodemailer, but if we want to have our own email domain we still need a dedicated mail server at some point (although I'd prefer having a mail server hosted for us so we don't have to worry about it, but DanVanAtta disagreed, because, well more expenses that can be avoided)